src/annotator/guest.js

```diff
@@ -146,7 +144,14 @@ export default class Guest extends Delegator {
       }
     });
 
-    this.plugins = {};
+    // Set the initial anchoring module. Note that for PDFs this is replaced by
+    // `PDFIntegration` below.
```

Comment:
The way that the `anchoring` property is set in PDFs is pretty confusing. The `anchoring` property is initially set to the HTML anchoring implementation and then `PDFIntegration` replaces it. I plan to replace this in future but didn't want to include too many changes in this PR.